v2.1/diagnostics-reporting.md

```diff
@@ -6,280 +6,32 @@ toc: true
-
-{{site.data.alerts.callout_info}}No actual table data or table/column names are shared, just metadata about the structure of tables. All names and other string values are scrubbed and replaced with underscores.{{site.data.alerts.end}}
-
-Detail | Description
--------|------------
-Table | Metadata about each table, such as its internal ID, when it was last modified, and how many times it has been renamed. Table names are replaced with underscores.
-Column | Metadata about each column in a table, such as its internal ID and type. Column names are replaced with underscores.
-Column Families | Metadata about [column families](column-families.html) in a table, such as its internal ID and the columns included in the family. Family and column names are replaced with underscores.
-Indexes | Metadata about the primary index and any secondary indexes on the table, such as the internal ID of an index and the columns covered by an index. All index, column, and other strings are replaced with underscores.
-Privileges | Metadata about user [privileges](privileges.html) on the table, such as the number of privileges granted to each user. Usernames are replaced with underscores.
-Checks | Metadata about any [`CHECK` constraints](check.html) on the table. `CHECK` constraint names and expressions are replaced with underscores.
-
-#### Example
-
-This JSON example shows an excerpt of what table structure details look like when sent to Cockroach Labs, in this case for a node with just one table. Note that all names and other strings have been scrubbed and replaced with underscores.
-
-~~~ json
-{
-   "schema":[
-      {
-         "name":"_",
-         "id":51,
-         "parent_id":50,
-         "version":1,
-         "up_version":false,
-         "modification_time":{
-            "wall_time":0,
-            "logical":0
-         },
-         "columns":[
-            {
-               "name":"_",
-               "id":1,
-               "type":{
-                  "kind":1,
-                  "width":0,
-                  "precision":0
-               },
-               "nullable":true,
-               "default_expr":"_",
-               "hidden":false
-            },
-            ...
-         ],
-         ...
-      }
-   ]
-}
-~~~
-
-### SQL query statistics
-
-Each node of a CockroachDB cluster shares the following statistics about the SQL queries it has executed on an hourly basis:
+When diagnostics reporting is on, each node of a CockroachDB cluster shares anonymized details on an hourly basis, including data about the following:
 
-{{site.data.alerts.callout_info}}No query results are shared, just the queries themselves, with all names and other strings scrubbed and replaced with underscores, and statistics about the queries.{{site.data.alerts.end}}
+- Stores on the node
+- Hardware the node is running on
+- Structure of tables stored on the node
+- SQL queries executed by the node
```

Comment:
Tiny nit: I might say ‘types of SQL queries...’  just to reinforce that they’re *not* the raw queries, just the structure. I know we also talk about the string scrubbing a few lines later, but lazy readers like me might be skimming or someone might see that line out of context and be alarmed.

Reply:
Done.